deploy/03_deploy_vending_machine.ts

```diff
@@ -0,0 +1,42 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments } = hre
+  const { deployer } = await getNamedAccounts()
+  const { read } = deployments
+
+  const KeepToken = await deployments.get("KeepToken")
+  const T = await deployments.get("T")
+
+  const keepTotalSupply = await read("KeepToken", "totalSupply")
+
+  const tTotalSupply = await read("T", "totalSupply")
+
+  const vendingMachine = await deployments.deploy("VendingMachine", {
+    from: deployer,
+    // We're wrapping 100% of the minted KEEP and will be allocating 45% of the
+    // minted T tokens. The remaining T tokens will be in the future distributed
+    // between another instance of the VendingMachine (which will be wrapping NU
+    // token) and a DAO treasury.
+    args: [
+      KeepToken.address,
+      T.address,
+      keepTotalSupply,
+      tTotalSupply.mul(45).div(100),
```

Comment:
Can this multiplication by `0.45` be handled in a better way? I don't like what I did here :D I tried to handle it in different ways, but everything else that I tried was not working correctly (I was getting overflow/underflow errors or the computed value was `0`)...

Reply:
It looks legit to me!

Reply:
But let's extract the value to a constant.

Reply:
Done, see 3c36d41687fb8e59a4b7632ef73abb3ed1a6f5ca.